server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec.scala

```diff
@@ -70,13 +72,16 @@ class ElasticsearchRepositorySpec extends WordSpec with SearchMatchers with Elas
   val hybridNagCustCode = "1110"
   val hybridNagCustName = "EXETER"
   val hybridNagCustGeogCode = "E07000041"
+  val hybridNagRelatives = "[]"
   // Fields with this value are not used in the search and are, thus, irrelevant
   val hybridNotUsed = ""
   val hybridNotUsedNull = null
 
   // Secondary PAF/NAG is used for single search (to have some "concurrence" for the main address)
   // and in the Multi Search
   val hybridSecondaryUprn = 2L
+  val hybridSecondaryParentUprn = 4L
+  val hybridSecondaryRelatives ="h03r"
```

Comment:
Relatives is actually an `Array[Map[String, Any]]`. As an example:
https://github.com/ONSdigital/address-index-data/blob/develop/batch/src/test/scala/uk/gov/ons/addressindex/utils/SqlHelperSpec.scala#L152